docs/installation/config_cli.rst

```diff
@@ -0,0 +1,135 @@
+===========
+
+Sites configuration
+-------------------
+
+Configure the domain where Objects API is hosted
+
+* ``SITES_CONFIG_ENABLE``: enable Site configuration. Defaults to ``True``.
+* ``OBJECTS_DOMAIN``:  a ``[host]:[port]`` or ``[host]`` value. Required.
+* ``OBJECTS_ORGANIZATION``: name of Objects API organization. Required.
+
+Objecttypes configuration
+-------------------------
+
+Objects API uses Objecttypes API to validate data against JSON schemas, therefore
+it should be able to request Objecttypes API.
+
+* ``OBJECTS_OBJECTTYPES_CONFIG_ENABLE``: enable Objecttypes configuration. Defaults
+  to ``True``.
+* ``OBJECTTYPES_API_ROOT``: full URL to the Objecttypes API root, for example
+  ``https://objecttypes.gemeente.local/api/v1/``. Required.
+* ``OBJECTTYPES_API_OAS``: full URL to the Objecttypes OpenAPI specification.
+* ``OBJECTS_OBJECTTYPES_TOKEN``: authorization token. Required.
+* ``OBJECTS_OBJECTTYPES_PERSON``: Objects API contact person. Required.
+* ``OBJECTS_OBJECTTYPES_EMAIL``: Objects API contact email. Required.
+
+Demo user configuration
+-----------------------
+
+Demo user can be created to check if Objects API work. It has superuser permissions,
+so its creation is not recommended on production environment.
+
+* ``DEMO_CONFIG_ENABLE``: enable demo user configuration. Defaults to the value of the ``DEBUG`` setting.
+* ``DEMO_PERSON``: demo user contact person. Required.
+* ``DEMO_EMAIL``: demo user email. Required.
+* ``DEMO_TOKEN``: demo token. Required.
+
+
+Objecttypes API
+===============
+
+ObjectTypes API has similar configuration steps as the Objects API.
+
+Sites configuration
+-------------------
+
+Configure the domain where Objects API is hosted
+
+* ``SITES_CONFIG_ENABLE``: enable Site configuration. Defaults to ``True``.
+* ``OBJECTTYPES_DOMAIN``:  a ``[host]:[port]`` or ``[host]`` value. Required.
+* ``OBJECTTYPES_ORGANIZATION``: name of Objecttypes API organization. Required.
+
+Objects configuration
+---------------------
+
+Objects API uses Objecttypes API to validate data against JSON schemas, therefore
+it should be able to request Objecttypes API.
+
+* ``OBJECTS_OBJECTTYPES_CONFIG_ENABLE``: enable Objecttypes configuration. Defaults
+  to ``True``.
+* ``OBJECTTYPES_API_ROOT``: full URL to the Objecttypes API root, for example
+  ``https://objecttypes.gemeente.local/api/v1/``. Required.
+* ``OBJECTTYPES_API_OAS``: full URL to the Objecttypes OpenAPI specification.
+* ``OBJECTS_OBJECTTYPES_TOKEN``: authorization token. Required.
+
+Demo user configuration
+-----------------------
+
+The similar configuration as in Objects API.
+
+* ``DEMO_CONFIG_ENABLE``: enable demo user configuration. Defaults to the value of the ``DEBUG`` setting.
+* ``DEMO_PERSON``: demo user contact person. Required.
+* ``DEMO_EMAIL``: demo user email. Required.
+* ``DEMO_TOKEN``: demo token. Required.
+
+
+Execution
+=========
+
+
+With the full command invocation, everything is configured at once and immediately
+tested.
```

Comment:
I don't think we have a readthedocs for setup-configuration yet, but maybe it would be nice to have a reference to that here at some point

Reply:
Good idea. For now setup_configuration lib has just README, I'll put a link for it here